camel/agents/task_agent.py

```diff
@@ -375,3 +376,103 @@ def run(
 
         sub_tasks_msg = task_response.msgs[0]
         return get_task_list(sub_tasks_msg.content)
+
+
+class RoleAssignmentAgent(ChatAgent):
```

Comment:
Add a test for this newly introduced class.

Reply:
Thanks, I created `examples\test\test_ai_society_with_role_generation.py`, but I am not so sure if I do right.

Reply:
Done